### DISCUSSION TITLE
Question about badges

### DISCUSSION THREAD

**Discussion by DeSn4ke** [aiko-chan-ai/DiscordBotClient#151] [Q&A]
Hello aiko chan, first of all i would thank you for your work to make a discord client bot it's just so cool, and second i wanted to know, when we all log into our bots, it shows that you are a staff member of discord with the badges, is it possible to get the same thing or is it special ? Just for fun so it's just visual, i just wanted to know that :3

**Comment by aiko-chan-ai** [comment]
that can be done by slightly changing the code

**Comment by DeSn4ke** [comment]
Thanks, now all i need to know is how i guess x)

**Comment by DeSn4ke** [comment]
Could you tell me where i need to change the code please or with what
